server/views/partials/expectedArrivalCardMacro.njk

```diff
@@ -39,6 +39,43 @@
     </dl>
 {% endmacro %}
 
+{% macro recentArrivalCardListContent(arrival, loopIndex) %}
```

Comment:
maybe these should go in `recentArrivalCardMacros`?

Reply:
sorted in latest PR